DAO.sol

```diff
@@ -814,6 +824,17 @@ contract DAO is DAOInterface, Token, TokenSale {
         return daoCreator.createDAO(_newCurator, 0, 0, now + splitExecutionPeriod);
     }
 
+    uint stackDepthControlTmp;
+
+    function stackDepthControl() returns (bool) {
+        if (stackDepthControlTmp++ < 16){
```

Comment:
Since `16` here is a constant that we choose, move it to the top of the interface with a descriptive name as a `uint const` like the others we are gathering there. Possible name:  `stackDepthCeiling`